### PR TITLE
perf(DS-368): add support for olive Open edX release

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
## Description

This PR adds support for the olive release maintaining the current support for nutmeg. It was tested on the two local environments.

## Testing instructions

Setting up the environment:
1. Install eox-core in LMS/LMS worker:
```
pip install git+https://github.com/eduNEXT/eox-core.git@MJG/DS-364-olive-compatibility#egg=eox_core==8.0.0_1
```
2. Install eox-audit-model: 
```
pip install git+https://github.com/eduNEXT/eox-audit-model.git@MJG/olive-support#egg=eox_audit_model==2.0.0_1
```
3. Migrate eox-audit-model: `python manage.py lms migrate eox_audit_model` (LMS)
4. Restart services
5. Add `CELERY_ALWAYS_EAGER = False` to your development settings.

Generating audit entries using eox-core:
1. Go to http://lms-url/eox-core/api-docs
2. Create a user and then create the same user for the 2nd time
3. Go to /admin and search for `audit`. You'll see the new record generated by the transaction.

## Additional information

Detailed test cases:
https://docs.google.com/document/d/1v6mxFSf1k7JrxU36Ok9uVDFBhtxVuGmhFLtxtbIsPFs

## Checklist for Merge

- [ ] Tested in a remote environment
- [x] Updated documentation
- [x] Rebased master/main
- [x] Squashed commits

### CODE REVIEW

**Comment by mariajgrimaldi** [comment]
Failing requests are not being registered. I'm looking into it

**Comment by MaferMazu** [comment]
I tried without the CELERY_ALWAYS_EAGER configuration, and it works well ✅

But when I set `CELERY_ALWAYS_EAGER = False`, the audit model didn't create an audit track, and I received this:
Screenshot from 2023-01-25 17-24-02

Do you have any idea what could be happening?

File: .github/workflows/bump_version.yml [hunk 1]
```diff
@@ -16,14 +16,14 @@ jobs:
           token: ${{ secrets.DEDALO_PAT }}
       - name: Get next version
         id: tag_version
-        uses: mathieudutour/github-tag-action@v6.0
+        uses: mathieudutour/github-tag-action@v6.1
```
Comment (MaferMazu) [inline .github/workflows/bump_version.yml:19]: Wouldn't it be better to use v6 without the minor version here?

**Comment by dcoa** [comment]
> Do you have any idea what could be happening? Does it happen to someone else, or is it just me?

I've been testing the plugin and works as expected, no errors.

**Review by dcoa — APPROVED** [review]

File: .github/workflows/bump_version.yml (hunk 1 above)
Comment (mariajgrimaldi) [reply .github/workflows/bump_version.yml:19]: It doesn't exist :/

**Comment by mariajgrimaldi** [comment]
@MaferMazu: can you include the complete traceback from the lms-worker?

**Comment by MaferMazu** [comment]
Here is the output from the lms-worker:

> 2023-01-27 01:03:59,953 ERROR 1 [celery.worker.consumer.consumer] [user None] [ip None] consumer.py:520 - Received unregistered task of type 'eox_audit_model.tasks.create_audit_register'.
> The message has been ignored and discarded.
> 
> Did you remember to import the module containing this task?
> Or maybe you're using relative imports?
> 
> Please see
> http://docs.celeryq.org/en/latest/internals/protocol.html
> for more information.
> 
> The full contents of the message body was:
> b'{"task": "eox_audit_model.tasks.create_audit_register", "id": "3e7356a2-e7e0-4b43-b702-f408f8ba3545", "args": [], "kwargs": {"action": "Create edxapp user", "status": 1, "method_name": "eox_core_api_method", "captured_logs": "2023-01-27 01:03:59,922 INFO 30 [edx.student] [user 4] [ip 172.21.0.1] management.py:518 - user 15 originated from a white labeled \\"Microsite\\"", "traceback_log": "NoneType: None\\n", "input_parameters": {"args": [{"email": "user12@example.com", "username": "string12", "fullname": "string", "password": "*****", "year_of_birth": "*****", "gender": "*****", "goals": "*****", "mailing_address": "*****", "level_of_education": "*****", "activate_user": "*****", "skip_password": "*****"}], "kwargs": {}}, "output_parameters": "<Response status_code=200, \\"text/html; charset=utf-8\\">", "site_id": 4, "performer": "admin", "ip": "172.21.0.1", "notes": null, "eox_tenant_sender": "lms.olive.edunext.link"}, "group": null, "group_index": null, "retries": 0, "eta": null, "expires": null, "utc":... (1118b)
> 
> Thw full contents of the message headers:
> {'eox_tenant_sender': 'lms.olive.edunext.link'}
> 
> The delivery info for this task is:
> {'exchange': '', 'routing_key': 'edx.lms.core.default'}
> Traceback (most recent call last):
>   File "/openedx/venv/lib/python3.8/site-packages/celery/worker/consumer/consumer.py", line 591, in on_task_received
>     strategy = strategies[type_]
> KeyError: 'eox_audit_model.tasks.create_audit_register'

**Review by MaferMazu — APPROVED** [review]
I could fix it with @mariajgrimaldi's help. We need to add eox-audit-model in INSTALLED_APPS, eox-core did it for us [here](https://github.com/eduNEXT/eox-core/blob/674a87c564d45be26524f7348e2283f8b0387396/eox_core/settings/common.py#L80). I installed eox-core in lms-worker and it's working.

This PR looks good to me ✨

**Pull request merged**